docs/source/diagnosis.md

```diff
@@ -0,0 +1,64 @@
+# Quantization diagnosis
+There are 2 ways to diagnose quantized model with Intel® Neural Compressor. First is non-GUI mode that is described below and second is GUI mode with [Neural Insights](https://github.com/intel/neural-compressor/tree/master/neural_insights) component.
+
+## Supported feature matrix
+<table class="center">
+    <thead>
+        <tr>
+            <th>Types</th>
+            <th>Quantization</th>
```

Comment:
I think this column could be "diagnosis data", so it could cover both quantize and benchmark.